types/react-native/globals.d.ts

```diff
@@ -18,3 +18,5 @@ declare function setImmediate(handler: (...args: any[]) => void): number;
 
 declare function cancelAnimationFrame(handle: number): void;
 declare function requestAnimationFrame(callback: (time: number) => void): number;
+
+declare function fetch(input: RequestInfo, init?: RequestInit): Promise<Response>;
```

Comment:
`RequestInfo`, `RequestInit`, `Response` are dom defination and not defined in react-native.

Reply:
it is indeed.

It seems that I should remove dom from tsconfig in all  projects related `react-native`

Reply:
And add `Websocket`.